cristin-commons/src/main/java/no/unit/nva/cristin/model/JsonPropertyNames.java

```diff
@@ -50,4 +50,19 @@ public class JsonPropertyNames {
     public static final String EMAIL = "email";
     public static final String PHONE = "phone";
 
+    public static final String PROJECT_MANAGER = "project_manager";
```

Comment:
Maybe this should have been refactored in another PR (this is some old code already present, not something you introduced) to use CristinQuery class or another new query param constants class as these values are not json field values but query parameters.

Reply:
I was slightly confused with that too.